README.md

```diff
@@ -33,3 +33,83 @@ Your pull requests are welcome but please keep them manageable and focussed.
 Equally your input on any pull requests we have in flight at any given time is invaluable to us, so please get involved.
 
 Thanks! :grin:
+
+
+## Running the example
+
+* Clone the repo
+* cd to `example` folder
+* run `flutter packages get` to install dependencies
+* `flutter run` will start the application on connected android / iOS device
+
+
+## Using the Realtime API
+
+##### Import the package
+```dart
+import 'package:ably_flutter_plugin/ably.dart' as ably;
+```
+
+##### Create an Ably instance
+```dart
+final ably.Ably ablyPlugin = ably.Ably();
```

Comment:
This does not match Ably's conventions in pretty much every other SDK. I don't follow why the user needs to define an instance of an Ably plugin at all, and deviate so significantly from other SDKs.

What this should look like is:

```
import 'package:ably_flutter_plugin/ably.dart' as Ably;
final Ably.Rest ably = Ably.Rest(key: 'key');
Ably.Rest.Channel channel = ably.channels.get('test');
channel.publish(name: 'event');
```

Reply:
I think we should be able to do that, but this requires some changes that are very critical to the complete plugin based approach.

Any comments from @QuintinWillison on this would be helpful!

Reply:
> I think we should be able to do that, but this requires some changes that are very critical to the complete plugin based approach.

Well the default position should be adherence to our API, unless there is a compelling reason to deviate.  Is there one?

Reply:
@mattheworiordan There is no reason, just that I'm reusing the same codebase developed by Quicktin in early days and parked the thought of optimizing it for later. However, I'd prefer to try and this handled in a separate PR as current PR is purely focused on proving Event handling.

cc: @QuintinWillison @paddybyers

Reply:
Thanks @tiholic - can you please create an issue to do that work under, bidirectionally link it and this conversation and then resolve this conversation.

Reply:
Issue logged https://github.com/ably/ably-flutter/issues/15